src/app/genes/gene-overview/soe/soe-chart/soe-chart.component.html

```diff
@@ -5,10 +5,12 @@
         <h2 class="chart-header">{{title.toUpperCase()}}</h2>
         <div class="spacer"></div>
         <i class="pi pi-window-maximize" *ngIf="geneScore" (click)="onExpand()"></i>
-        <infobuttonpopover
-            [contentProps]="{ 'ownerId': wikiInfo.ownerId, 'wikiId': wikiInfo.wikiId }"
-            >
-        </infobuttonpopover>
+        <img src="/assets/icon/info-icon3.svg" class="info-icon pointer" (click)="op.show($event, actualTarget)"/>
+        <p-overlayPanel #op>
+            <ng-template pTemplate>
+                <wiki [ownerId]="wikiInfo.ownerId" [wikiId]="wikiInfo.wikiId"></wiki>
+            </ng-template>
+        </p-overlayPanel>
```

Comment:
Now using PrimeNG's [overlay panel](https://www.primefaces.org/primeng/v9-lts/#/overlaypanel) since we're using that library already and it's more reliable than the React one. This fixes the bug where the popovers would appear at the top of the page.